.openpublishing.publish.config.json

```diff
@@ -72,7 +72,7 @@
     {
       "path_to_root": "articles/quantum",
       "url": "https://github.com/microsoft/quantum",
-      "branch": "master",
+      "branch": "feature/standalone",
```

Comment:
```suggestion
      "branch": "master",
```

Reply:
We should adopt this suggestion, but **only** once the corresponding PR from `feature/standalone` → `master` has been merged on the samples.